sycl/source/detail/graph_impl.hpp

```diff
@@ -367,7 +364,7 @@ class exec_graph_impl {
   /// Turns the internal graph representation into UR command-buffers for a
   /// device.
   /// @param D Device to create backend command-buffers for.
```

Comment:
Know this wasn't something you've changed but noticed it here:
```suggestion
  /// @param Device Device to create backend command-buffers for.
```